core/iwasm/aot/aot_loader.c

```diff
@@ -1169,17 +1176,30 @@ load_init_expr(const uint8 **p_buf, const uint8 *buf_end, AOTModule *module,
             read_uint64(buf, buf_end, expr->u.i64);
             break;
         case INIT_EXPR_TYPE_V128_CONST:
-            read_byte_array(buf, buf_end, &expr->u.v128, sizeof(expr->u.v128));
+            i64x2 = (uint64 *)expr->u.v128.i64x2;
+            CHECK_BUF(buf, buf_end, sizeof(uint64) * 2);
+            wasm_runtime_read_v128(buf, &i64x2[0], &i64x2[1]);
+            buf += sizeof(uint64) * 2;
             break;
         case INIT_EXPR_TYPE_GET_GLOBAL:
             read_uint32(buf, buf_end, expr->u.global_index);
             break;
+#if WASM_ENABLE_GC == 0
```

Comment:
Had better be `#if WASM_ENABLE_GC == 0 && WASM_ENABLE_REF_TYPES != 0` like other places.